app/app.go

```diff
@@ -127,9 +127,9 @@ import (
 
 	// Note: please do your research before using this in production app, this is a demo and not an officially
 	// supported IBC team implementation. It has no known issues, but do your own research before using it.
-	// intertx "github.com/cosmos/interchain-accounts/x/inter-tx"
```

Comment:
I will keep inter-tx commented out until official supported versions are available. This is just a TODO